code/modules/fish/fishtank.dm

```diff
@@ -239,6 +237,13 @@
 //		SUPPORT PROCS		//
 //////////////////////////////
 
+/obj/machinery/fishtank/proc/get_num_fish()
+	var/fish_count = 0
+	for(var/fish in fish_list)
```

Comment:
Why is a for loop needed here? Can't you just do a `return length(fish_list)`?
Wouldn't even need the proc if that's the case. Unless you think it makes it more clear.